go.mod

```diff
@@ -32,8 +32,11 @@ require (
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	knative.dev/pkg v0.0.0-20230712131115-7051d301e7f4
 	sigs.k8s.io/controller-runtime v0.17.2
+	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20240215124517-56159419231e
```

Comment:
nit: How did this end up getting added in as a requirement?

Reply:
my bad :) removed.